repos/tap/components/list/treeList/treeList.js

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback } from 'react'
+import { pickKeys, noPropArr } from '@keg-hub/jsutils'
+import { useTheme } from '@keg-hub/re-theme'
+import { useSelector, shallowEqual } from 'react-redux'
+import { setFeatureActive } from 'SVActions/features'
+import { loadFeature } from 'SVActions/features/loadFeature'
+import {
+  View,
+  Loading,
+  Text,
+} from 'SVComponents'
+import { Values } from 'SVConstants'
+import TreeView from 'react-native-final-tree-view'
+import { ChevronDown } from 'SVAssets/icons'
+import { useStoreItems } from 'SVHooks/store/useStoreItems'
+
+const { CATEGORIES } = Values
+
+/**
+ * Finds the node based on given id
+ * @param {string} id 
+ * @param {Array<object>} nodes 
+ * 
+ * @returns {object} node
+ */
+const findNode = (id, nodes) => {
+  let foundNode = {}
+  nodes.reduce((__, node) => {
+    if (node.id === id) foundNode = node
+    if (node.children.length > 0) {
+      findNode(id, node.children)
+    }
+  }, {})
+
+  return foundNode
+}
+
+/**
+ * TreeList
+ * @param {Object} props 
+ */
+export const TreeList = props => {
+
+  const { features, fileTree=noPropArr } = useStoreItems([CATEGORIES.FEATURES, CATEGORIES.FILE_TREE])
+
+  const onItemPress = useCallback(({node}) => {
+    // for now only supporting feature file/content
```

Comment:
This is fine, but the `content` property should exist on all test file types. I made an update last week so that they all have the `content` property